src/system-image.js

```diff
@@ -55,7 +55,6 @@ var installLatestVersion = options => {
       .waitForDevice()
       .then(() => adb.wipeCache())
       .then(() => utils.log.debug("adb wiped cache"))
-      .then(() => adb.shell("mount -a"))
```

Comment:
we then need to mount the 3 needed ones, since this was added my me long time ago since not all the devices mounted this by default *cough* legacy devices *cough*

Reply:
Ah ok I see. maybe mount -a -F (forked mounting) might suppress the errors. Its just that mount -a has no "ignore errors" option which is dumb...